vtStor.ATest/vtStor.ATest.vcxproj

```diff
@@ -72,22 +72,22 @@
   <PropertyGroup Label="UserMacros" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
     <LinkIncremental>true</LinkIncremental>
-    <OutDir>$(ProjectDir)$(Platform)$(Configuration)\</OutDir>
+    <OutDir>$(SolutionDir)$(Platform)$(Configuration)\</OutDir>
```

Comment:
I change it to test directly, if not changing, it needs to copy the output file into the destination to run. I discussed on it with anh Tam.